localisation/00_wc_sad.csv

```diff
@@ -694,23 +694,23 @@ EVTDESC_WCCSO.10202_learning;As time progresses, the whispers get stronger and t
+whispers_option_learning;�Y[Root.GetFirstName]�! will focus on writing down whispers about �YLearning�!.;;�Y[Root.GetFirstName]�! konzentriert sich auf das Niederschreiben von Einfl�sterungen �ber �YBildung�!.;;;;;;;;;;;x
 nick_the_corruptor;the Corruptor;;[From.Getderdie] Verderber[From.GetinFemale];;;;;;;;;;;x
 text_effect_qiraji_retinue;Enables �GSilithid Swarm�! retinue that is normally available to �YQiraji�! culture only;;Aktiviert das �GSilithiden-Schwarm�!-Gefolge f�r die �YQiraji�!-Kultur;;;;;;;;;;;x
 text_effect_naga_retinue;Enables �GRoyal Guard�! retinue that is normally available to �YNaga�! culture only;;Aktiviert das �GK�nigswache�!-Gefolge f�r die �YNaga�!-Kultur;;;;;;;;;;;x
 text_effect_titanforged_retinue;Enables �GTitan-forged Legion�! retinue that is normally available to �YTitan-forged�! culture group only;;Aktiviert das �GTitanische Legion!-Gefolge f�r die �Ytitanische�! Kultur;;;;;;;;;;;x
-whispers_tooltip_tip_1;It's common that Twillight Hammer cultists can hear the whispers of the Old Gods, but especially enthusiastic individuals write them down in order to preserve the enigmatic wisdom of their masters to show their �Y[This.TrueReligion.GetPietyName]�!. However, because they often do not understand what they do, the results of their efforts have the �Ylowest quality�!.;;;;;;;;;;;;;x
-whispers_tooltip_tip_2;The Old Gods' slaves that are widely known for their �Y[This.TrueReligion.GetPietyName]�! frequently hear the whispers of their master, but being able to decipher them is quite �YUncommon�!.;;;;;;;;;;;;;x
-whispers_tooltip_tip_3;It's exceptionally �YRare�! for the Old Gods to grant mortal zealots not only the ability to hear their sacred voices for their �Y[This.TrueReligion.GetPietyName]�!, but also understand what they say to some extent.;;;;;;;;;;;;;x
-whispers_tooltip_tip_4;The Old Gods whisper myths of long forgotten times, a blessing that only the most devoted get for their �Y[This.TrueReligion.GetPietyName]�!. If someone writes these revelations down, [This.GetHerHis] efforts will bring the �Yhighest quality�! results.;;;;;;;;;;;;;x
+whispers_tooltip_tip_1;It's common that cultists of the Twillight Hammer can hear the whispers of the Old Gods, but especially enthusiastic individuals write them down in order to preserve the enigmatic wisdom of their masters to show their �Y[This.TrueReligion.GetPietyName]�!. However, because they often do not understand what they do, the results of their efforts have the �Ylowest quality�!.;;Es ist �blich, dass Kultisten des Schattenhammers die Einfl�sterungen der alten G�tter h�ren k�nnen, aber besonders eifrige Individuen schreiben sie auf, um die r�tselhafte Weisheit ihrer Meister zu bewahren und ihre �Y[This.TrueReligion.GetPietyName]� zu zeigen! Da sie jedoch oft nicht verstehen, was sie tun, sind die Resultate ihrer Bem�hungen von �Yniedrigster Qualit�t�!.;;;;;;;;;;;x
```

Comment:
```suggestion
whispers_tooltip_tip_1;It's common that cultists of the Twilight Hammer can hear the whispers of the Old Gods, but especially enthusiastic individuals write them down in order to preserve the enigmatic wisdom of their masters to show their �Y[This.TrueReligion.GetPietyName]�!. However, because they often do not understand what they do, the results of their efforts have the �Ylowest quality�!.;;Es ist �blich, dass Kultisten des Schattenhammers die Einfl�sterungen der alten G�tter h�ren k�nnen, aber besonders eifrige Individuen schreiben sie auf, um die r�tselhafte Weisheit ihrer Meister zu bewahren und ihre �Y[This.TrueReligion.GetPietyName]� zu zeigen! Da sie jedoch oft nicht verstehen, was sie tun, sind die Resultate ihrer Bem�hungen von �Yniedrigster Qualit�t�!.;;;;;;;;;;;x
```

Reply:
Oh, nice you spotted the double l in Twilight. I think, no-one else saw that :D fixed now